### ISSUE TITLE
Besti 9 now supports Buttplug <3

### ISSUE THREAD

**Issue by WeaselKing** [buttplugio/awesome-buttplug#21]
Heyo! I'm happy to announce that Silly Hoofprint LLC's Besti 9 now supports Buttplug.io. Should have done it sooner because it's awesome. We're using velocity to adjust general vibration, and special events also trigger animated patterns of interaction with the toy which get added to whatever other vibration is going on (things like the AI having an orgasm and the like.) Settings to adjust it are in there, too, like what types of things make vibration and how intense it is. As soon as the scalar communication thing goes in I'll be able to add squeezy functions, too <3

Huge thanks for your work on this it's a breeze to program against.

https://besti.love is the website.

**Comment by qdot** [comment]
Hah, beat you to it, I added Besti yesterday after the contact from Ko-Fi. You're at the top of the games list now! :)

Scalar is done in the dev branch, I'm trying to get sensors in then we'll be shipping it, I'm *hoping* we're talking weeks at this point.

I'd be happy to discuss methods for stroking toys too, if you're interested in integrating those!

**Comment by WeaselKing** [comment]
Awesome! Thanks so much <3 It's a really neat feature to add and I'm just
thrilled with how it works. I'm super excited for the scalar thing!

I'm very interested in integrating something for stroking toys! Is that an
upcoming feature, or can that be done now? What's an example toy that does
that? I'll order one and integrate it.

I'm currently working on an original piece of hardware for Besti X so that
would be a fun thing to write firmware appropriate that can talk to
Buttplug. Chip shortage is a PITA but I got the parts I need yesterday and
my silicone production facility's done, too so as soon as the next major
release is out (same timeframe as you, "Weeks" which is god knows when
Unity's involved :P) I can continue on that properly. I'm not sure how
often you don't have to reverse engineer a communication protocol with a
device, but it might make it nice when I can just make the firmware meet
you someplace useful <3 I went to university for electrical engineering so
it's nice to use that training on something physical.

--Skunk/Patrick
